components/user/card-activity.vue

```diff
@@ -7,21 +7,27 @@
 			<!-- assets -->
 			<div class="relative self-start">
 				<img
-					:src="`https://cdn.discordapp.com/app-assets/${data.application_id}/${data.assets!.large_image}.png`"
-					:alt="data.assets!.large_text"
-					width="60"
+					v-if="spotifyImageUrl" 
+					:src="spotifyImageUrl" 
+					:alt="data.assets!.large_text" 
+					width="60" 
 					height="60"
+					class="block rounded-lg object-cover" 
+					:class="{ 'large-mask': data.assets!.small_text }" />
+				<img 
+					v-if="data.assets!.large_text && !spotifyImageUrl"
```

Comment:
You can use `v-else-if`:
```suggestion
					v-else-if="data.assets!.large_text"
```